src/commands/cluster-replicate.json

```diff
@@ -14,8 +14,30 @@
         ],
         "arguments": [
             {
-                "name": "node-id",
-                "type": "string"
+                "name": "args",
+                "type": "oneof",
+                "arguments": [
+                    {
+                        "name": "node-id",
+                        "type": "string"
+                    },
+                    {
+                        "name": "no-one",
+                        "type": "block",
+                        "arguments": [
+                            {
+                                "name": "no",
+                                "type": "pure-token",
+                                "token": "NO"
+                            },
+                            {
+                                "name": "one",
+                                "type": "pure-token",
+                                "token": "ONE"
+                            }
```

Comment:
In `REPLICAOF ip port | REPLICAOF NO ONE`, it is two arguments in both cases, like host=NO port=ONE, so it is the same arity in both cases.

For `CLUSTER REPLICATE (node-id | NO ONE)`, NO ONE replaces the node-id, so the arity is 3 or 4. Need to change `arity: 3` in line 7 of this JSON file. Or maybe should use a single token instead of NO ONE? For example NONE?

Reply:
Updated artity field. I'm not sure what this field is for. Since it can be calculated from ```arguments``` field. Also everything works even with ```arity=3```. I think for users similarity to ```REPLICAOF NO ONE``` is better than trying to keep the same arity for different cases. I think client are not thinking about arity at all, but for us it is easier to distinguish <node-id> from a special case.

Reply:
OK.

I don't know why we have arity here but it's returned by COMMAND and COMMAND INFO, which I think some clients use.

We can use a negative number `-3`. It means *3 or more*.